tests/fixture_valid.js

```diff
@@ -308,6 +311,35 @@ module.exports.checkFixture = function checkFixture(fixture, usedShortNames=[])
   return result;
 };
 
+function checkCoarseChannelExistence(result, fixture, fineChannels, switchingChannels, ch, mode, chNumber) {
+  const coarseChannelKey = fineChannels[ch];
+
+  // the mode must also contain the coarse channel
+  if (!mode.channels.includes(coarseChannelKey)) {
```

Comment:
I spotted exactly this issue when I manually added the horizon-8 fixture for testing purposes. Thanks anyway!